curvefs/src/mds/fs_manager.cpp

```diff
@@ -843,6 +843,18 @@ void FsManager::RefreshSession(const RefreshSessionRequest* request,
 
     // update this client's alive time
     UpdateClientAliveTime(request->mountpoint(), request->fsname());
+    FsInfo* fsinfo;
+    FsInfoWrapper wrapper;
+    FSStatusCode ret = fsStorage_->Get(request->fsname(), &wrapper);
+    if (ret != FSStatusCode::OK) {
+        LOG(WARNING) << "GetFsInfo fail, get fs fail, fsName = "
+                    << request->fsname()
+                    << ", errCode = " << FSStatusCode_Name(ret);
+        return;
+    }
+
+    *fsinfo = wrapper.ProtoFsInfo();
```

Comment:
*fsinfo not initialized, here temporary variable fsinfo here is not necessary
`response->set_enablesumindir(wrapper.ProtoFsInfo().enablesumindir());` or Add method GetEnableSumInDir in class FsInfoWrapper and can use directly like `response->set_enablesumindir(wrapper.GetEnableSumInDir())`